{{cookiecutter.project_slug}}/compose/production/traefik/traefik.toml

```
@@ -0,0 +1,44 @@
+logLevel = "ERROR"
+defaultEntryPoints = ["http", "https"]
+
+[web]
+# Port for the status page
+address = ":8080"
+dashboard = true
+
+# Entrypoints, http and https
+[entryPoints]
+  # http should be redirected to https
+  [entryPoints.http]
+  address = ":80"
+    [entryPoints.http.redirect]
+    entryPoint = "https"
+  # https is the default
+  [entryPoints.https]
+  address = ":443"
+    [entryPoints.https.tls]
+
+# Enable ACME (Let's Encrypt): automatic SSL
+[acme]
+# Email address used for registration
+email = "{{ cookiecutter.email }}"
+storageFile = "/etc/traefik/acme/acme.json"
+entryPoint = "https"
+onDemand = false
+OnHostRule = true
+  # Use a HTTP-01 acme challenge rather than TLS-SNI-01 challenge
+  [acme.httpChallenge]
+  entryPoint = "http"
+
+[file]
+[backends]
+  [backends.django]
+    [backends.django.servers.server1]
+      url = "http://django:5000"
+
+[frontends]
+  [frontends.django]
+    backend = "django"
+    passHostHeader = true
```

To keep the change from #1852 we need to whitelist the `X-CSRFToken` to proxy it.

I dug a bit in the documentation, and it seems it's done like this:

```suggestion
    passHostHeader = true
    [frontends.django.headers]
      HostsProxyHeaders = ['X-CSRFToken']
```

Source: [security headers documentation](https://docs.traefik.io/basics/#security-headers) which points at [unrolled/secure for the possible secure headers](https://github.com/unrolled/secure#default-options).